tests/resources/test-number.json

```diff
@@ -25,5 +25,11 @@
     "zNumberLetter": "Z",
     "sequenceNumber": 1000000,
     "testNumberKey": 5
+  },
+  {
+    "tNumber": "020000T",
```

Comment:
is it supposed to start with "0" ?

Reply:
It does not matter as long as it is in this format: `nnnnnnT`.